src/groupeddataframe/groupeddataframe.jl

```diff
@@ -380,7 +380,19 @@ function Base.iterate(gd::GroupedDataFrame, i=1)
     end
 end
 
-Compat.lastindex(gd::GroupedDataFrame) = gd.ngroups
```

Comment:
Why did we need Compat before and not now?

Reply:
probably it was needed before Julia 1.0. I have checked that `lastindex` is defined in Julia 1.0.